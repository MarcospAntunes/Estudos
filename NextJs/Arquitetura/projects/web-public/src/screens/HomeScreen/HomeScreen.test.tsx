import { customRender } from "@alura/test-commons/react-testing-library";
import HomeScreen from "./HomeScreen";

const render = customRender((props) => {
  return <section>{props.children}</section>;
});

describe("<HomeScreen />", () => {
  it("renders the page", () => {
    const { container } = render(<HomeScreen />);

    expect(container).toMatchSnapshot();
  });
});
