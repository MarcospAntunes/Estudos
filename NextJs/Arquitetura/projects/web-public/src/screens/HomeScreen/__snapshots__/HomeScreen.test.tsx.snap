// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders the page 1`] = `
<div>
  <section>
    <main>
      <h1
        class="rwbqK"
      >
        Home
      </h1>
      <p
        class="rwbqK"
      >
        Importando modulo local: @alura/utils/math/sum sum(2,2): 
        4
      </p>
    </main>
  </section>
</div>
`;
